src/main/java/com/unascribed/fabrication/client/FabricationConfigScreen.java

```diff
@@ -999,7 +993,7 @@ private int drawConfigValue(MatrixStack matrices, String key, String title, Stri
 		float scale = 1;
 		boolean noUnset = key.startsWith("general.");
 		ConfigValue currentValue = noUnset ? (isEnabled(key) ? ConfigValue.TRUE : ConfigValue.FALSE) : onlyBannable ? getValue(key) == ConfigValue.BANNED ? ConfigValue.BANNED : ConfigValue.UNSET : getValue(key);
-		boolean keyEnabled = isEnabled(key);
+		boolean keyEnabled = getResolvedValue(key) == ResolvedConfigValue.DEFAULT_TRUE;
```

Comment:
This will only be true if the setting is `unset` and is implicitly `true`, which differs from the original implementation (`isEnabled` will be true if the setting is set to `true` explicitly as well.)

Did you mean to check `.value`?

Reply:
since this is supposed to show the parent value checking .value (which is what the original isEnabled call did) produces the wrong result with how things are at the moment